toolz/itertoolz.py

```diff
@@ -496,6 +496,34 @@ def concatv(*seqs):
     return concat(seqs)
 
 
+def duplicate(seq, n=1):
```

Comment:
This function would be more curryable if it accepted `n` first, this would make it easier to map over a sequence of sequences. Also, I don't think we need a default for `n` because not passing `n` is just a nop.